recipes/crowcpp-crow/all/conanfile.py

```diff
@@ -16,13 +17,15 @@ def _source_subfolder(self):
 
     def requirements(self):
         self.requires("boost/1.75.0")
-        '''To be removed in next released versions'''
+        # To be removed in next released versions.
         self.requires("openssl/1.1.1i")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        extracted_dir = "crow-" + self.version
-        os.rename(extracted_dir, self._source_subfolder)
+        tools.get(
```

Comment:
Please add `required_conan_version = ">=1.33.0"` at the top of the file since this is a "new" feature.